kbfsblock/id.go

```diff
@@ -158,6 +158,16 @@ func MakePermanentID(encodedEncryptedData []byte) (ID, error) {
 	return ID{h}, nil
 }
 
+// MakePermanentIDV2 computes the permanent ID of a block given its
+// encoded and encrypted contents using v2 encryption.
+func MakePermanentIDV2(encodedEncryptedData []byte) (ID, error) {
```

Comment:
same comment about renaming the existing V1 function to have a `V1` suffix, and maybe having a `MakePermanentID(..., version)` function.

Reply:
Will try to fix in a cleanup PR, but note that the "version" here is different from the other places, as it refers to the hash version rather than the encryption version.  I'm not sure I want to explicit pass around the SHA constants in other parts of the code unless it's really necessary.  But maybe it is...